packages/client/src/comp/contact/ContactInfo.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { ContactData, Locale, ImplementedState } from '../../common'
 import styled from 'styled-components'
-import { ContactModal } from './ContactModal'
+import { ElectionOfficialPicker } from './ElectionOfficialPicker'
```

Comment:
Can we call it a `ContactPicker`?  Picker is more descriptive but by convention, the election official is a contact in our nomenclature.